src/org/labkey/test/util/APIUserHelper.java

```diff
@@ -238,4 +248,79 @@ protected void _deleteUsers(boolean failIfNotFound, String... userEmails)
                 deleteUser(userId);
         }
     }
+
+    private static final Pattern regEmailVerification = Pattern.compile("verification=([A-Za-z0-9]+)");
+    @Override
+    public String setInitialPassword(String email)
```

Comment:
Kind of interesting/funny that this functionality (for APIHelper) had been missing for all this time and wasn't needed until this story.

Reply:
I only figured out how to do this via API relatively recently to [create the initial user via API](https://github.com/LabKey/testAutomation/blob/develop/src/org/labkey/test/util/ApiBootstrapHelper.java#L174). It wasn't explicitly necessary for this story but one of the scenarios I was considering involved creating 100 users and setting that many passwords via UI would have taken a long time.